sickbeard/scene_exceptions.py

```diff
@@ -320,7 +320,7 @@ def _xem_exceptions_fetcher():
             if not parsedJSON['data']: 
                 logger.log(u"No data returned from XEM when checking scene exceptions. Update failed for {0}".format
                             (sickbeard.indexerApi(indexer).name), logger.DEBUG)
-                continue
+                break
```

Comment:
But that doesn't mean, you can't get the same result, when it just doesn't have results. You have to be careful with assumptions you make.